airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -69,3 +79,46 @@ def get_assets(
         assets=[AssetResponse.model_validate(asset, from_attributes=True) for asset in assets],
         total_entries=total_entries,
     )
+
+
+@assets_router.get(
+    "/events",
+    responses=create_openapi_http_exception_doc([401, 403, 404]),
+)
+async def get_asset_events(
+    limit: QueryLimit,
+    offset: QueryOffset,
+    order_by: Annotated[
+        SortParam,
+        Depends(
+            SortParam(
+                ["timestamp", "source_dag_id", "source_task_id", "source_run_id", "source_map_index"],
+                AssetEvent,
+            ).dynamic_depends("timestamp")
```

Comment:
I think we should remove "timestamp" here as there is no default set in the docs: https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html#operation/get_dataset_events

Reply:
I saw [this](https://github.com/apache/airflow/blob/main/airflow/api_connexion/endpoints/asset_endpoint.py#L124) in legacy code due to which I made this default. Also, noticed when I remove default from timestamp its considering `id` as default

Reply:
Resolving this conversation as legacy is behaving in same way